_posts/2021-04-09-tests-gives-you-confidence.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: "Tests gives you confidence - But data gives you power"
+description: ""
+category:
+tags: [python, pycon.il, elasticsearch, pytest]
+---
+{% include JB/setup %}
+
+# Tests gives you confidence - But data gives you power
+
+## Who Am I
+
+* Israel Fruchter, I'm working in testing for the past 15 year. previusly on the television and home entertiment domain, and nowdays at scylladb.
+
+## What's broken about test reporting
+
+As we all know, most of the testing frameworks we use and love, were written with the intention of running those quick and very precise unit test
+
+But then reality come knocking, products evolves, teams expend, microservices spawns. And you find yourself with silly amount of those functional or integration or selenium based tests, with run runtime of 30secs to 1 hour each, that you are running almost 24/7. Lots information get spit at back you, and you want to collect it put some order into it. see it clearly.
```

Comment:
"run runtime" :laughing: better use "run time"

Reply:
The description of the problem is a bit vague. too many tests and results? too many runs? the relation between test results not clear?
also, from the next part i understand - but it should be explained more clearly - that part of the problem is that test reports do not show you the time dimension, what happened to tests over time

Reply:
If you can tell what you understood from the next part, it might help me capture it a bit better